core/src/commonMain/kotlin/cache/data/GuildData.kt

```diff
@@ -65,7 +65,8 @@ public data class GuildData(
     val stageInstances: Optional<List<StageInstanceData>> = Optional.Missing(),
     val stickers: Optional<List<StickerData>> = Optional.Missing(),
     val guildScheduledEvents: Optional<List<GuildScheduledEventData>> = Optional.Missing(),
-    val premiumProgressBarEnabled: Boolean
+    val premiumProgressBarEnabled: Boolean,
+    val safetyAlertsChannelId: Snowflake?,
```

Comment:
It's weird this is required to be filled, according to how the docs describe it this should be optional

Reply:
in the guild object it's described to only be nullable: https://discord.com/developers/docs/resources/guild#guild-object-guild-structure (`?` before the type means nullable)